addons/controllers/testdata/test-vsphere-cpi-paravirtual.yaml

```diff
@@ -18,3 +18,22 @@ metadata:
 spec:
   vsphereCPI:
     mode: vsphereParavirtualCPI
```

Comment:
Could we add a testcase for fetching from `kube-system/kube-apiserver-lb-svc` as well? Since these functions are being adopted from guest-cluster-controller we should add similar test coverage here as well.

Reply:
Yes, will do.